src/metricscollector/server/memory_metric_handler.go

```diff
@@ -76,14 +76,14 @@ func (h *MemoryMetricHandler) GetMemoryMetricHistory(w http.ResponseWriter, r *h
 	if len(startParam) == 1 {
 		start, err = strconv.ParseInt(startParam[0], 10, 64)
 		if err != nil {
-			h.logger.Error("get-memory-metric-history-parse-start-time", err, lager.Data{"start": startParam})
+			h.logger.Error("get-memory-metric-histories-parse-start-time", err, lager.Data{"start": startParam})
```

Comment:
Shouldn't I have the appId?
I would create a local logger using WithData that contains the appId so it is automatically added everywhere.